SU-SWS/stanford_profile_helper
# D8CORE-3540: fixing the caption styling on the gallery.

# READY FOR REVIEW

# Summary
- Updating the caption style to match what is on the media caption
- changed the alignment to left for the sm-md breakpoint because it was misaligned at the right. 

# Needed By (Date)
- 3/4 if time

# Urgency
- low

# Steps to Test

1. Pull in this change
2. Clear caches
3. Look at a gallery with captions.
4. Notice they align right and have a different font size/color with a padding top.
5. Notice no changes to the ones without captions.

# Affected Projects or Products
- Gallery

# Associated Issues and/or People
- D8CORE-3540

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
